src/Analyzers/Core/Analyzers/AbstractCodeQualityDiagnosticAnalyzer.cs

```diff
@@ -48,16 +48,13 @@ public bool OpenFileOnly(SimplifierOptions? options)
             bool isUnnecessary,
             bool isEnabledByDefault = true,
             bool isConfigurable = true,
-            LocalizableString? description = null)
-#pragma warning disable RS0030 // Do not used banned APIs
```

Comment:
The CI error is because you are removing this pragma warning. Please restore it and CI should be green.